src/common.ts

```diff
@@ -0,0 +1,70 @@
+import { addMinutes, format, set } from 'date-fns';
+import type { z } from 'zod';
+
+export function formatDate(date: Date): string {
+  return format(date, 'yyyy-MM-dd');
+}
+
+/**
+ * @param date ISO 8601 date string
+ * @returns formatted matching Abax API date format
+ */
+export function formatDateTime(date: Date): string {
+  return date.toISOString();
+}
```

Comment:
I'm a bit confused. Where did we define date formatting before? It's not clear from the diff here why we have two different date formatting functions now.

Reply:
I thought it used to be a function in the utils file, but apparently it isn't. I guess we aren't using formatting often enough to warrant a function for it. We create a one for the yyyy-MM-dd like 2 places in the code so far. If we were to expand, having a function to standardise formatting would make sense then.